config/ingest.py

```diff
@@ -19,11 +21,13 @@
 
 config.register.unique = ['visit']
 config.register.columns = {
-    'visit': 'int',
-    'basename': 'text',
-    'filter': 'text',
-    'date': 'text',
-    'expTime': 'double',
-    'object': 'text',
+    'visit':      'int',
+    'basename':   'text',
+    'filter':     'text',
+    'date':       'text',
+    'expTime':    'double',
+    'object':     'text',
+    'imgType':    'text',
+    'wavelength': 'double',
 }
```

Comment:
Understood. I shall un-align everything to use a single space so that people aren't tempted to keep this going in future.